msal/src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -169,13 +169,15 @@ protected void ValidateScopeInput(SortedSet<string> scopesToValidate)
             catch (MsalException ex)
             {
                 apiEvent.ApiErrorCode = ex.ErrorCode;
-                AuthenticationRequestParameters.RequestContext.Logger.Error(ex);
+                string noPiiMsg = MsalExceptionFactory.GetPiiScrubbedExceptionDetails(ex);
```

Comment:
I believe there is an overload of Error that takes in an exception. It could be modified to get the scrubbed details as well.

Reply:
@bgavrilMS That's a good idea. I think we might do some refactoring of the logging, so that's something to consider. thanks.